android/app/build.gradle

```diff
@@ -0,0 +1,49 @@
+plugins {
+    id 'com.android.application'
+    id 'org.jetbrains.kotlin.android'
+}
+
+android {
+    namespace 'au.com.robin.sms'
+    compileSdk 34
+
+    defaultConfig {
+        applicationId "au.com.robin.sms"
+        minSdk 24
+        targetSdk 34
```

Comment:
I have an android phone that's either ice cream sandwich, or jelly bean? Can you make it work? Please comment add a comment about which sdk version is for what android version, and why we are on that version, perhaps a particular feature we need that only a specific sdk version has. A link to a page with all the SDK versions is good too.

Reply:
I may be able to try that out to support API >= 14. 

As for this 
> Please comment add a comment about which sdk version is for what android version, and why we are on that version, perhaps a particular feature we need that only a specific sdk version has. A link to a page with all the SDK versions is good too.

Do you want to have comments somewhere in that file explaining that a certain SDK is for a this Android version and the link to that page?

Reply:
yes please